tests/test_open.py

```diff
@@ -301,3 +302,50 @@ def test_islice_gzip_does_not_fail(tmp_path):
     f = dnaio.open(path)
     next(iter(f))
     f.close()
+
+
+def test_unsupported_mode():
+    with pytest.raises(ValueError) as error:
+        _ = dnaio.open(os.devnull, mode="x")
+    error.match("Mode must be")
+
+
+def test_no_file2_with_multiple_args():
+    with pytest.raises(ValueError) as error:
+        _ = dnaio.open(os.devnull, os.devnull, file2=os.devnull)
+    error.match("more than one unnamed file")
```

Comment:
Also needs to be adjusted if you accept my suggestion about changing the error message